docs/src/humann.md

```diff
@@ -1,4 +1,613 @@
+chocophlan : DEMO = http://huttenhower.sph.harvard.edu/humann_data/chocophlan/DEMO_chocophlan.v296_201901b.tar.gz
+uniref : uniref50_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref50_annotated_v201901b_ful
+l.tar.gz
+uniref : uniref90_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref90_annotated_v201901b_ful
+l.tar.gz
+uniref : uniref50_ec_filtered_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_ec_filtered/uniref50_ec_filte
+red_201901b_subset.tar.gz
+uniref : uniref90_ec_filtered_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_ec_filtered/uniref90_ec_filte
+red_201901b_subset.tar.gz
+uniref : DEMO_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref90_DEMO_diamond_v201901b.tar.
+gz
+utility_mapping : full = http://huttenhower.sph.harvard.edu/humann_data/full_mapping_v201901b.tar.gz
+```
+
+For example, if you'd like to install these databases to `/BigDrive/humann/`,
+you could run
+
+```julia-repl
+shell> humann_databases --download cholophlan full /BigDrive/humann/chocophlan
+# ... lots of output
+
+shell> humann_databases --download uniref uniref90_diamond /BigDrive/humann/uniref
+# ... lots of output
+
+shell> humann_databases --download utility_mapping full /BigDrive/humann/utility_mapping
+# ... lots of output
+```
+
+At some point, I'll write some functions to automate this, but for now,
+doing this will update a configuration file, so you shouldn't have to worry about it again.
+
+## Running HUMAnN
+
+[Official tutorial link](https://github.com/biobakery/biobakery/wiki/humann3#2-metagenome-functional-profiling)
+
+Some example files you can use to run this tutorial are available from the MetaPhlAn repo,
+and can be downloaded using the `Downloads` standard library in julia:
+
+```julia-repl
+julia> using Downloads: download
+
+julia> base_url = "https://github.com/biobakery/humann/raw/master/examples/";
+
+julia> files = [
+           "demo.fastq.gz",
+           "demo.sam",
+           "demo.m8"
+       ];
+
+julia> for file in files
+           download(joinpath(base_url, file), file)
+       end
+
+julia> readdir()
+5-element Vector{String}:
+ "Manifest.toml"
+ "Project.toml"
+ "demo.fastq.gz"
+ "demo.sam"
+ "demo.m8"
+```
+
+For convenience, this package has the [`humann()`](@ref) function,
+which can be used in your julia scripts to build and call the `humann` command line tool.
+
+For example, rather than call
+
+```sh
+$ humann --input demo.fastq.gz --output demo_fastq
+```
+
+You can do
+
+```julia-repl
+julia> humann("demo.fastq.gz", "demo_fastq")
+[ Info: Running command: humann -i demo.fastq.gz -o demo_fastq
+Creating output directory: /home/kevin/my_project/demo_fastq
+Output files will be written to: /home/kevin/my_project/demo_fastq
+Decompressing gzipped file ...
+```
+
```

Comment:
When I run this in my repl I get the error: `ERROR: failed process: Process(`humann -i demo.fastq.gz -o demo_fastq`, ProcessExited(1)) [1]` and I think it has something to do with installing bowtie2. However, when I try to install bowtie2 using conda or brew I get this error `Warning: No available formula or cask with the name "bowtie2"`

Not sure where to go from here...

Reply:
Did you use the `install_deps()` function, or some other process to install humann?

Reply:
I did `pip install humann`

Reply:
What happens if you do `which bowtie2` in the terminal? There's [this weird thing](https://www.biostars.org/p/494922/) with bowtie2 when you do installation through `conda`, but not sure if you're having a similar problem. Can you see what happens if you use `BiobakeryUtils.install_deps()`?

Reply:
Doing `which bowtie2` doesn't do anything. 
![Screen Shot 2021-10-10 at 7 04 33 PM](https://user-images.githubusercontent.com/60850811/136715731-8dc4c0ae-b5ff-457e-84a2-78eb63301d26.png)

Actually, when I tried installing with `conda` I get: `bash: conda: command not found` even though I have `conda` installed.